core/runtime/binaryen/module/module_instance_impl.cpp

```diff
@@ -92,4 +92,10 @@ namespace kagome::runtime::binaryen {
     return outcome::success();
   }
 
+  outcome::result<void> ModuleInstanceImpl::addToTls(
```

Comment:
Name confuses a bit, because inside only resetting of deleter

Reply:
Will refactor